scantools/qr/detector.py

```diff
@@ -0,0 +1,171 @@
+import csv
```

Comment:
This folder should maybe go into `proc/` ?

Reply:
Good, I moved to `proc/qrcode`.